python/ray/_raylet.pyx

```diff
@@ -324,6 +321,16 @@ class StreamingObjectRefGenerator:
                 raise StopIteration
         return ref
 
+    async def suppress_exceptions(self, ref: ObjectRef):
+        # Wrap a streamed ref to avoid asyncio warnings about not retrieving
+        # the exception when we are just waiting for the ref to become ready.
+        # The exception will get returned (or warned) to the user once they
+        # actually await the ref.
+        try:
+            await ref
```

Comment:
Should we use `exception()` instead here? https://docs.python.org/3/library/asyncio-task.html#asyncio.Task.exception

This way, we don't need to await on actual value if there's no exception

Reply:
`asyncio.wait` will await on the actual value anyway right now.